--- src/lxpolkit.c	2010-03-28 21:05:10.000000000 +0200
+++ src/lxpolkit.c.new	2010-03-28 21:27:16.513866888 +0200
@@ -37,7 +37,7 @@
 
 void show_msg(GtkWindow* parent, GtkMessageType type, const char* msg)
 {
-    GtkWidget* dlg = gtk_message_dialog_new(parent, GTK_DIALOG_MODAL, type, GTK_BUTTONS_OK, msg);
+    GtkWidget* dlg = gtk_message_dialog_new(parent, GTK_DIALOG_MODAL, type, GTK_BUTTONS_OK, "%s", msg);
     const char* title = NULL;
     switch(type)
     {
